mlir/test/Dialect/Affine/constant-fold.mlir

```diff
@@ -60,3 +60,22 @@ func.func @affine_min(%variable: index) -> (index, index) {
   // CHECK: return %[[r]], %[[C44]]
   return %0, %1 : index, index
 }
+
+// -----
+
+func.func @affine_apply_poison_division_zero() {
+  %c0 = arith.constant 0 : index
+  %0 = affine.apply affine_map<(d0)[s0] -> (d0 mod s0)>(%c0)[%c0]
+  %1 = affine.apply affine_map<(d0)[s0] -> (d0 floordiv s0)>(%c0)[%c0]
+  %2 = affine.apply affine_map<(d0)[s0] -> (d0 ceildiv s0)>(%c0)[%c0]
```

Comment:
This makes sense to me, the symbol could come from some other simplification and it would be better not to trigger a verifier failure